src/components/hero.js

```diff
@@ -1,20 +1,26 @@
 import React from 'react';
 
+import bg from '../static/hero.jpg';
+
 const Hero = () => {
+  const background = {
+    backgroundImage: `url(${bg})`,
+    width: '100%',
+    height: '500px',
+  }
+
   return (
-    <div class="pt-24">
-      <div class="container px-3 mx-auto flex flex-wrap flex-col md:flex-row items-center">
-        <div class="flex flex-col w-full md:w-2/5 justify-center items-start text-center md:text-left">
-          <p class="uppercase tracking-loose w-full">What business are you?</p>
-          <h1 class="my-4 text-5xl font-bold leading-tight">Main Hero Message to sell yourself!</h1>
-          <p class="leading-normal text-2xl mb-8">Sub-hero message, not too long and not too short. Make it just right!</p>
-          <button class="mx-auto lg:mx-0 hover:underline bg-white text-gray-800 font-bold rounded-full my-6 py-4 px-8 shadow-lg">Subscribe</button>
-        </div>
-        <div class="w-full md:w-3/5 py-6 text-center">
-          <img class="w-full md:w-4/5 z-50" src="http://steamavatars.co/wp-content/uploads/ayeeeeee-steam-avatars.jpg" />
+    <React.Fragment>
+      <div class="bg-cover" style={background}>
```

Comment:
React will render when you use `class` however it will produce a warning. It's best to use `className` to reduce the noise in the console.

```suggestion
      <div className="bg-cover" style={background}>
```